README.md

```diff
@@ -223,6 +223,10 @@ files after saving by using the
 have the plugin, just add `let g:syntastic_haml_checkers = ['haml_lint']` to
 your `.vimrc`.
 
+### Vim 8 / Neovim
```

Comment:
While I also question the value of these paragraphs, I think it's okay to cross-link right now. If we end up having too many of these, I might go ahead and remove them, move them to a separate page, or summarize them in some way.